src/logger.ts

```diff
@@ -14,13 +14,13 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-const https = require('https');
+import * as https from 'https';
 
-let baseUrl;
-let mxAccessToken;
-let mxRoomId;
+let baseUrl: string;
+let mxAccessToken: string;
+let mxRoomId: string;
 
-function setup(matrixServer, roomId, accessToken) {
+function setup(matrixServer: string, roomId: string, accessToken: string) {
```

Comment:
can't get at the line, but types on the error, warn, info, debug proxy functions please

Reply:
Okay, added types, though the type is `any[]`.